### ISSUE TITLE
can i specify channel in metric reporter in config file

### ISSUE THREAD

**Issue by habout632** [facebookresearch/pytext#230]
seems that we can't do this now

**Comment by Titousensei** [comment]
That's correct, by default the metric reporter emits to all relevant channels. Please elaborate: what's your use case and an example of what you would like to do?
